examples/volumes.py

```diff
@@ -38,44 +38,130 @@
 config = try_load_from_file(config)
 
-# To run this example you must have created a volume on OneView and defined here its id
-volume_id = ''
+# To run this example, you may set a WWN to create a volume using the WWN of the volume (optional)
+unmanaged_volume_wwn = ''
 
-options = {
-    "name": "Test Snapshot",
-    "description": "Description for the snapshot",
-    "snapshotType": "PhysicalCopy"
+oneview_client = OneViewClient(config)
+
+# Defines the storage system and the storage pool which are provided to create the volumes
+storage_system = oneview_client.storage_systems.get_all()[0]
+storage_pools = oneview_client.storage_pools.get_all()
+storage_pool_available = False
+for sp in storage_pools:
+    if sp['storageSystemUri'] == storage_system['uri']:
+        storage_pool_available = True
+        storage_pool = sp
+if not storage_pool_available:
+    raise ValueError("ERROR: No storage pools found attached to the storage system")
+
+# Create a volume with a Storage Pool
+print("\nCreate a volume with a specified Storage Pool")
+
+options_with_storage_pool = {
+    "name": 'ONEVIEW_SDK_TEST_VOLUME_TYPE_1',
+    "description": 'Test volume with common creation: Storage Pool',
+    "provisioningParameters": {
+        "provisionType": 'Full',
+        "shareable": True,
+        "requestedCapacity": 1024 * 1024 * 1024,  # 1GB
+        "storagePoolUri": storage_pool['uri']
+    }
 }
+volume_with_storage_pool = oneview_client.volumes.create(options_with_storage_pool)
+pprint(volume_with_storage_pool)
 
-oneview_client = OneViewClient(config)
+# Create a volume with a Storage System, a Storage Pool, and a Snapshot Pool
+print("\nCreate a volume with a specified Snapshot Pool")
+
+options_with_snapshot_pool = {
+    "name": 'ONEVIEW_SDK_TEST_VOLUME_TYPE_3',
+    "description": 'Test volume with common creation: Storage System + Storage Pool + Snapshot Pool',
+    "provisioningParameters": {
+        "provisionType": 'Full',
+        "shareable": True,
+        "requestedCapacity": 1024 * 1024 * 1024,  # 1GB
+        "storagePoolUri": storage_pool['uri']
+    },
+    "storageSystemUri": storage_system['uri'],
+    "snapshotPoolUri": storage_pool['uri']
+}
+volume_with_snapshot_pool = oneview_client.volumes.create(options_with_snapshot_pool)
+pprint(volume_with_snapshot_pool)
+
+# Create a volume for management by the appliance using the WWN of the volume
+print("\nCreate a volume for management by the appliance using the WWN of the volume")
+
+if unmanaged_volume_wwn:
+    options_with_wwn = {
+        "type": "AddStorageVolumeV2",
+        "name": 'ONEVIEW_SDK_TEST_VOLUME_TYPE_4',
+        "description": 'Test volume with common creation: Storage System + Storage Pool + WWN',
+        "storageSystemUri": storage_system['uri'],
+        "wwn": unmanaged_volume_wwn,
+        "provisioningParameters": {
+            "shareable": False
+        }
+    }
+    volume_created_with_wwn = oneview_client.volumes.create(options_with_wwn)
```

Comment:
Maybe this should be called volume_added_with_wwn instead of volume_created_with_wwn